### PR TITLE
Feature/id 31/jwt authorization

### PR DESCRIPTION
**Pull request by victor-fundacion**
Clickup: https://app.clickup.com/t/868ah1knj

### CODE REVIEW

**Comment by CatCoder36** [comment]
Please add a description toRename the folder name of this service to EventService and add a description to your user story

**Comment by victor-fundacion** [comment]
> Please add a description toRename the folder name of this service to EventService and add a description to your user story @victor-fundacion

@CatCoder36, First, the rename of foder was not consulted to the team at the time your commet was posted, and overall, it is out of this User Story scope. Rember it is about Authorization, and the service you want me to rename has been worked on for almost 2 weeks. And Secondly, The user story had previous description and acceptance criteria (you can check the timeline). But, you are right, I forgot to add it to my PR's description. Thxs.

**Review by Elvis-Castro-H — CHANGES_REQUESTED** [review]
For testing reasons I protected the endpoint to get all the events
image

When I tried to do a get without being logged in this was allowed
image
image

File: apps/cevent_frontend/src/app/my_tickets/page.tsx [hunk 1]
```diff
@@ -1,7 +1,36 @@
+'use client'
+import { redirect, useRouter } from 'next/navigation';
+import { routes } from '@/utils/navigation/Routes';
+import { Button } from "@mui/material";
+import { useSession } from "next-auth/react";
+import { useEffect, useState } from "react";
+
 export default function MyTicketsPage() {
```
Comment (CatCoder36) [inline apps/cevent_frontend/src/app/my_tickets/page.tsx:8]: Why does my ticket page currently require me to check the authentication status?

File: apps/cevent_frontend/src/utils/interfaces/EventInterfaces.ts [hunk 2]
```diff
@@ -24,3 +24,36 @@ export interface EventDetailDto extends EventHomePageDto {
   organizerUserId: number;
   createdAt: Date;
 }
+
+interface Event {
```
Comment (CatCoder36) [inline apps/cevent_frontend/src/utils/interfaces/EventInterfaces.ts]: Please consider not replacing the above existing event interfaces with a more general one.

File: apps/docker-compose.yml [hunk 3]
```diff
@@ -68,13 +68,10 @@ services:
 
   # SQL Server
   sqlserver:
-    image: mcr.microsoft.com/mssql/server:2022-latest
-    container_name: sqlserver_container
+    image: mcr.microsoft.com/mssql/server:2019-latest
```
Comment (CatCoder36) [inline apps/docker-compose.yml:71]: Why replace sql server 2022 with 2019?

**Review by CatCoder36 — APPROVED** [review]

**Review by Elvis-Castro-H — APPROVED** [review]

**Pull request merged**